### ISSUE TITLE
Asset Browser keyboard navigation stops working after adding entity

### ISSUE THREAD

**Issue by niklas-ourmachinery** [OurMachinery/themachinery-public#354]
1. Open animation project.
2. Select an asset in the Asset Browser and use keyboard keys to move around.
3. Right-click -> New Entity -> press Return

Note that keyboard navigation doesn't work any more.

**Comment by simon-ourmachinery** [comment]
This is what I did:

1. opened the animation project
2. selected a random asset moved around
3. right click -> New Entity -> pressed Return

![assetb](https://user-images.githubusercontent.com/70658708/99248321-8e77d700-2808-11eb-947f-f6705b5aa6bc.gif)


I can reproduce this in the released version but not in the internal development version. This is why I am closing this issue.
